src/ray/stats/metric.cc

```diff
@@ -41,6 +43,12 @@ void RegisterAsView(opencensus::stats::ViewDescriptor view_descriptor,
   view_descriptor.RegisterForExport();
 }
 
+static inline int _grpc_opencensus_initializer = []() {
+  grpc::RegisterOpenCensusPlugin();
```

Comment:
Will we have ip or node id label so that we know which node emits these grpc metrics?

Reply:
added the global tags in exporter.